apps/src/music/views/Callouts.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {useSelector} from 'react-redux';
+import {MusicState} from '../redux/musicRedux';
+import moduleStyles from './callouts.module.scss';
+const FontAwesome = require('../../templates/FontAwesome');
+
+const availableCallouts: {
+  [key: string]: {selector: string; arrowAngle: number};
+} = {
+  'play-sound-block': {
+    selector: ".blocklyFlyout g[data-id='play-sound-block']",
+    arrowAngle: 0,
+  },
+  'when-run-block': {
+    selector: "g[data-id='when-run-block'] > path",
+    arrowAngle: 0,
+  },
+  'run-button': {selector: '#run-button', arrowAngle: 180},
+};
+
+/**
+ * Renders one of several pre-defined callouts.
+ */
+const Callouts: React.FunctionComponent = () => {
```

Comment:
Should this be in /lab2 and be made available for all lab2 labs?

Reply:
That's probably a good thing to do, but I might hold off until we flesh this version out a bit more first.  I'm concerned that using CSS selectors, even when driven by a specific lab, isn't the most elegant solution.  It might make sense that some labs will be able to render callouts in the right places by having their internal UI components render the callouts as part of the component render (imagine a button rendering an attached callout, along with a highlight on the button itself), which would affect the design of this system.

Reply:
Yeah that's fair. Mind leaving a comment/TODO around this (either here or in musicRedux?)

Reply:
Done.